tests/integration/cattletest/core/test_whitelist_default_registries.py

```diff
@@ -140,7 +142,7 @@ def test_registries(super_client, client, context,
 
     env = _create_stack(client)
 
-    uuid = "sim:namespace/ubuntu:{}".format(random_num())
+    uuid = "sim:namespace.com/repo/ubuntu:{}".format(random_num())
     lc_registry = {"imageUuid": uuid}
```

Comment:
But `sim:namespace/ubuntu:123` should still process right? In that scenario, "namespace" would be the registry, right?